scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -157,6 +158,7 @@
 
 logger.info(f"SF file: {args.sfFile}")
 
+##Fixme later..PU helper is now default for 2018
```

Comment:
This comment seems to be lying, the default is still 2016, unless it means something else